chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -555,11 +555,11 @@ public void streamSEDDetailsDocumentContents(final HttpServletResponse response)
     }
 
     @ApiOperation(value = "Update an existing certified product.",
-            notes = "Updates the certified product after first validating the request. The logged in"
-                    + " user must have ROLE_ADMIN, ROLE_ONC, or ROLE_ACB and have administrative "
-                    + " authority on the ACB that certified the product. If a different ACB is passed in"
-                    + " as part of the request, an ownership change will take place and the logged in "
-                    + " user must have ROLE_ADMIN or ROLE_ONC.")
+            notes = "Updates the certified product after first validating the request. If a different "
+                    + "ACB is passed in as part of the request, an ownership change will take place and "
+                    + "the logged in user must have ROLE_ADMIN or ROLE_ONC.  Security Restrictions: ROLE_ADMIN, "
+                    + "ROLE_ONC, or ROLE_ACB and have administrative authority on the ACB that certified "
```

Comment:
Can a listing be transferred from one ACB to another by a user that is ROLE_ACB? If so, do they need ACB admin on both ACBs? If so, that's not explicit here. It looks like they only need it on the originating ACB

Reply:
Updated the description to match the manager's security.